yb-voyager/src/issue/dml.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright (c) YugabyteDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package issue
+
+var advisoryLocksIssue = Issue{
+	Type:            ADVISORY_LOCKS,
+	TypeName:        "Advisory Locks",
```

Comment:
why do we need Type and TypeName both?

Reply:
https://github.com/yugabyte/yb-voyager/pull/1873#discussion_r1837759390

Reply:
yeah went through that thread, but still didn't what are the usecases where we need both.
Is this the only difference - "replacing _ in Type with whitespaces for TypeName" @makalaaneesh

Reply:
Just separation of concerns. Type is the enum. TypeName is for display purposes.
In `cmd` layer, when we need to filter issues based on their types, it feels wrong to me to do them by their names (i.e. strings with spaces). Even in json, for the type, we should be sending the type (enum, without white spaces), and not the type name.

Reply:
type name is just for html report?

Reply:
Yep. Good point, I should probably remove `TypeName` from here then, no? This can be the headache of the assessmentReport layer 🤔

Reply:
yeah, extra handling might be a headache. I think we can live with `_` in html report, or do some hardcoding where required.

Reply:
I am not sure if it can always be the case that `TypeName` will be the string replacing underscores with whitespace from `Type.` Maybe in some cases, we want to keep the Type very minimal and TypeName a bit descriptive, e.g., `unsupported_indexes`, `Indexes on complex datatypes`?